pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -312,6 +311,26 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 	return managedresources.CreateForSeed(ctx, b.client, b.namespace, managedResourceControlName, false, resources)
 }
 
+func getDruidDeployCommands(config *config.GardenletConfiguration) []string {
+	command := []string{"" + "/bin/etcd-druid"}
+	command = append(command, "--enable-leader-election=true")
+	command = append(command, "--ignore-operation-annotation=false")
+	command = append(command, "--workers=50")
+	command = append(command, "--compaction-workers=3")
+
+	if config != nil {
+		if config.ETCDConfig != nil {
```

Comment:
Nit: This could be moved into one line to reduce the nesting. However, see above proposal, then it will be one condition anyways.

Reply:
Removed the nesting